src/pretix/control/forms/filter.py

```diff
@@ -739,7 +739,6 @@ class VoucherFilterForm(FilterForm):
         label=_('Search voucher'),
         widget=forms.TextInput(attrs={
             'placeholder': _('Search voucher'),
-            'autofocus': 'autofocus'
```

Comment:
Did you remove this on purpose?

Reply:
I did, as the commit message states: [control] Remove second autofocus element on voucher list

Reply:
Sorry, I didn't look at the commit message.